unlock-app/src/hooks/useTermsOfService.js

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react'
+
+export const localStorageKey = 'terms-of-service'
+
+/**
+ * This hook retrieves metadata for a token
+ * @param {*} address
+ */
+export const useTermsOfService = () => {
```

Comment:
Thanks for encapsulating the localstorage use here, that's a good pattern we should follow in the future!